manifests/rhoai/params.env

```diff
@@ -0,0 +1 @@
+odh-training-operator-controller-image=docker.io/kubeflow/training-operator:v1-855e096
```

Comment:
@astefanutti is there an quay image for this ?

Reply:
Not yet. We can keep the upstream one in the short term until we have automated build for the container image.

Reply:
Ok.